designs/virtualization/4.7/add-os-image-updates/add-os-image-to-common-template 4.7.md

```diff
@@ -0,0 +1,32 @@
+---
+parent: Virtualization
+version: 4.7
+---
+# Add an operating system image to a common template
+
+## About
+
+This is an update for the 4.6 version.
+This will cover an update for:
+- the "Operating system" dropdown
+- "Source" dropdown name, placement and options
+
+## OS dropdown
+
+![os dropdown](img/Admin-create-4.7-0-1.jpg)
+The dropdown options will be ordered by two sections, promoting the options with an available image above the ones without.
+This dropdown will also have a search field to avoid looking for a specific OS in two different lists.
```

Comment:
I wonder if we need the search. Will we have so many options that it will get used?

Reply:
If you need windows 10 and currently that option does not have an image, then the process might go like this

The user scrolls for windows in the first "Available source" section of the drop-down,
spots the "windows" options but not "10",
The user might choose other windows under the impression that there is no "Windows 10" option.
If the user do see that there is an "Unavailable source" section, he will need to start the search again.

I think having a search will help in both scenarios.
The search might be type-ahead within the field itself. That is better.

Reply:
search can't hurt either I suppose